api/src/main/java/ca/bc/gov/educ/grad/report/api/config/ReportServiceConfiguration.java

```diff
@@ -40,7 +45,9 @@ public ModelMapper modelMapper() {
     public WebClient webClient() {
         HttpClient client = HttpClient.create();
         client.warmup().block();
-        return WebClient.builder().build();
+        return WebClient.builder()
```

Comment:
Please add a log() function as per other apis. You can see how others use it here: https://github.com/bcgov/EDUC-GRAD-GRADUATION-API/blob/main/api/src/main/java/ca/bc/gov/educ/api/graduation/config/RestWebClient.java#L77